knot.schema.json

```diff
@@ -610,6 +610,16 @@
             }
           ]
         },
+        "redundant-cast": {
+          "title": "detects redundant `cast` calls",
```

Comment:
Should this be capitalized? I saw some rule titles are capitalized and others are not.

Reply:
If we're already inconsistent, then I think it doesn't matter for this PR; we need to pick a preference and run through them to make them consistent either way. I'm not sure what's better at the moment.